src/command-line/list.js

```diff
@@ -1,8 +1,8 @@
 "use strict";
 
-var ClientManager = new require("../clientManager");
-var program = require("commander");
-var colors = require("colors/safe");
+const colors = require("colors/safe");
+const program = require("commander");
+const fs = require("fs");
 const Helper = require("../helper");
```

Comment:
@astorije You may have forgot to move this into callback?

Reply:
I can move it into the callback, but I believe it's safe in here, no?
In fact, all `const Helper = require("../helper");` but one were at the top of the file, except one which was in a function, I think. Any reason to not have them at the top (aside your explanation on `ClientManager`, which makes sense, but it shouldn't apply for `Helper`).

Reply:
This PR before rebase had all helpers inside callbacks, after the rebase this one specifically was left out, while others are still in the callback, for consistency sake, requires that run our should live in callbacks (to prevent future problems).

Reply:
Sorry, I'm confused. I'd like to double-check with you before committing any further :)

All `src/command-line/*` files (after this PR, in the current state) but 2 (`remove` and `reset`) have the `require("../helper")` statement at the top and not in the callback.
I don't get why we should put `require("../helper")` in the callback, imo they should really be at the top of each file.

Let me know what you want me to do.

Reply:
Okay, keep them at the top for now.